README.md

```diff
@@ -20,15 +20,19 @@ https://user-images.githubusercontent.com/19291722/208880256-b0da924e-254e-4bc4-
 You can download the binary for your system in the [releases page](https://github.com/iits-consulting/otc-auth/releases).
 Unpack the binary and add it to your PATH and you are good to go!
 
+### Arch Linux
+
+Users of Arch Linux may use the package published in the [AUR](https://aur.archlinux.org/packages/otc-auth)
+
```

Comment:
Can we maybe automate this in our release pipeline? Then we can always make sure, that on the AUR is an up to date version of this programm.

Reply:
That is certainly possible. I will look into providing a job for including it into the release pipeline.